### PR TITLE
[#1] Setup project

### LINKED ISSUE #1: Setup project
## Why

- Create a new empty project for development (Suggestion: Generate new project from [Coroutine Template](https://github.com/nimblehq/android-templates/tree/develop/CoroutineTemplate))

- Update README.md
- Update default repo's url in `.github` templates

## Who Benefits?

Developers

### PR DESCRIPTION
**Pull request by hoangnguyen92dn**
Closes https://github.com/nimblehq/jetpack-compose-crypto/issues/1

## What happened 👀

Setup project ready for developer

## Insight 📝

- Generate new project from [Coroutine Template](https://github.com/nimblehq/android-templates/tree/develop/CoroutineTemplate)

- Update README.md

- Update default repo's url in `.github` templates

## Proof Of Work 📹

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 
 [LICENSE]: /LICENSE
 
 ## About
-<a href="https://nimblehq.co/">
-  <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png">
-    <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-160.png">
-  </picture>    
-</a>
```
Comment (luongvo) [inline README.md]: I think we would like this keep this 🤓

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luongvo) [inline app/src/main/res/layout/fragment_home.xml]: Should we also remove XML layout setup here? Or you're planning to remove it in a new PR?

**Review by luongvo — COMMENTED** [review]
rest lgtm

File: app/src/main/java/co/nimblehq/compose/crypto/model/UserUiModel.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.compose.crypto.model
+
+import co.nimblehq.compose.crypto.domain.model.User
+
+data class UserUiModel(
```
Comment (Wadeewee) [inline app/src/main/java/co/nimblehq/compose/crypto/model/UserUiModel.kt]: Do we need this file? Should we remove it 🤔

**Comment by Wadeewee** [comment]
@hoangnguyen92dn I think we can remove directory `second` in `screen` directory or what do you think?

**Comment by Wadeewee** [comment]
I thought we might be remove `BaseFragment` and `BaseFragmentCallbacks` as well as we have `BaseComposeFragment` and `BaseComposeFragmentCallbacks` 🤔

File: app/src/main/java/co/nimblehq/compose/crypto/model/UserUiModel.kt (hunk 3 above)
Comment (Wadeewee) [reply app/src/main/java/co/nimblehq/compose/crypto/model/UserUiModel.kt]: Same for `UserResponse` and `UserRepositoryImpl` in module `:data`, Also all of related with `user` in module `:domain` 🙏

**Comment by Wadeewee** [comment]
Rest LGTM 🚀

**Comment by hoangnguyen92dn** [comment]
@luongvo @Wadeewee The initial idea was to generate the project from our Coroutine template, then we could remove unused files in development. But I changed my mind with your suggestion, I will remove unnecessary files in this PR and make it clean and ready for development

**Comment by hoangnguyen92dn** [comment]
@luongvo @Wadeewee Updated 🚀

**Review by Wadeewee — APPROVED** [review]

**Review by luongvo — APPROVED** [review]
lgtm

**Pull request merged**